taxa/profile/tpdesceditor.php

```diff
@@ -244,7 +247,7 @@
 		}
 	}
 	else{
-		echo '<h2>No descriptions available.</h2>';
+		echo '<h3>' . $LANG['NO_DESC'] . '</h3>';
```

Comment:
If changing h2 to h3, make sure there's an h2 elsewhere in the file. Otherwise, Total Validator Pro will get mad. If the issue is that h2 font-size is too big, use styling to change it. E.g., `style="font-size: 1rem;"` or whatever looks good. taxa/profile/tpdesceditor.php already has a section:
```
<style>
		fieldset{ width:90%; margin:10px; padding:10px; }
		legend{ font-weight: bold }
	</style>
```

where you could add styling classes to just this file.